config/schema.yaml

```diff
@@ -927,6 +929,105 @@ schema:
         - Tract.skymap
         - Tract.tract
 
+    CalibrationLabel:
+      doc: >
+        A join table that associates an instrument and label with a date range,
+        used to associate master calibration products with exposures.
+      limited: false
+      columns:
+      -
+        name: instrument
+        type: string
+        length: 8
+        primary_key: true
+        nullable: false
+        doc: >
+          Name of the instrument associated with these master calibration
+          products.
+      -
+        name: calibration_label
+        type: string
+        length: 128
+        primary_key: true
+        nullable: false
+        doc: Label used to identify a group of master calibration products.
+      -
+        name: valid_first
+        type: datetime
+        doc: >
+          TAI timestamp of first exposure included in the range (inclusive).
+          May be far in the past to indicate an open interval.
+      -
+        name: valid_last
+        type: datetime
+        doc: >
+          TAI timestamp of last exposure included in the range (inclusive).
+          May be far in the future to indicate an open interval.
+      foreignKeys:
+      -
+        src: instrument
+        tgt: Instrument.instrument
+
+    ExposureCalibrationLabelJoin:
```

Comment:
Yes, it definitely is, and having it just appear in the unit tests makes sense; this should now behave pretty much just like any of the spatial join views (which also should appear by name only in the tests), but is computed from temporal overlap rather than spatial overlap.